components/dash-core-components/tests/integration/calendar/test_date_picker_single.py

```diff
@@ -71,6 +73,11 @@ def test_dtps010_local_and_session_persistence(dash_dcc):
     assert dash_dcc.get_logs() == []
 
 
+@pytest.mark.xfail(
```

Comment:
Is there a pattern to which tests you marked `xfail` and which you marked `skipif`?

Reply:
I skipped those that were guaranteed to fail, xfail for those that can pass, but I added the xfail after reruns failed on dcc but it wasn't actually set, so I think now they will xpass most of the time.